DuckDuckGo/DaxDialogViewController.swift

```diff
@@ -125,13 +125,25 @@ class DaxDialogViewController: UIViewController {
     }
     
     private func updateMessage() {
+        let theme = self.theme ?? ThemeManager.shared.currentTheme
+        
         guard let message = message else { return }
-        label.attributedText = attributedString(from: String(Array(message)[0 ..< position]))
+        let baseText = attributedString(from: String(Array(message)[0 ..< position]), color: theme.daxDialogTextColor)
+        label.attributedText = baseText
+        
+        let restOfString = String(Array(message)[position...])
+        guard let nextWhitespace = restOfString.firstIndex(where: { $0.isWhitespace }) else { return }
+        let restOfWord = String(restOfString[restOfString.startIndex ..< nextWhitespace])
+        let invisible = attributedString(from: restOfWord, color: theme.daxDialogBackgroundColor)
+        
```

Comment:
Let me know if there's a more optimal way to do this.  Feels like string manipulation like this should be easier 🙄